src/datastructures.jl

```diff
@@ -316,39 +353,30 @@ end
 
 penalty(s::AbstractMinMaxConstraint, t) = s.penalty[t]
 
-""" A regulated hydropower reservoir, modelled as a `Storage` node.
+"""
+    HydroReservoir{T} <: EMB.Storage{T}
+
+A regulated hydropower reservoir, modelled as a `Storage` node. A regulated hydro storage node
+requires a storage volume for the `vol` and volume inflow `vol_inflow`. The `stor_res` is
+typically water in million cubic meters. Minimum, maximum and schedule volume constraints
+can be added using `Data` input of the composite types `MinConstraint`, `MaxConstraint` and
+`ScheduleConstraint`. These are given relative sizes between 0 and 1 relative to the total
+storage volume `vol`
 
```

Comment:
Resource har vel ikke enhet: The stored resource `stor_res` is typically water. Vurder å heller ha med typiske enheter på volume (million cubic meters) og inflow (million cubic meters per time unit eller cubic meters/sec)?

Reply:
Yes, resources do not have a unit.

However, that implies that we have to be careful with respect to conversion factors. These should be as close to 1 as possible to avoid introducing large matrix coefficient, and hence, a large condition number.